serverbase/source/served/serverbase.d

```diff
@@ -540,7 +545,6 @@ mixin template LanguageServerRouter(alias ExtensionModule, LanguageServerConfig
 
 		fibers.put("RPC manager", rpc);
 
-		spawnFiberImpl = (&pushFiber!(void delegate())).toDelegate;
```

Comment:
undo the remove, it's used here: https://github.com/Pure-D/serve-d/blob/1c7a2676c6451b4aaf1d08cd3551619d9aa7a2bf/serverbase/source/served/utils/async.d#L24

Reply:
I'm sorry, I didn't understand the code and the test didn't provide me with any hints.